src/GraphQLParser.Tests/GraphQLParser.Tests.csproj

```diff
@@ -11,11 +11,12 @@
 
   <ItemGroup>
     <PackageReference Include="Shouldly" Version="4.0.0-beta0002" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.0" />
```

Comment:
Added explicitly since it was removed from `xunit.runner.visualstudio` dependencies.